neo/Consensus/ConsensusService.cs

```diff
@@ -74,6 +74,18 @@ private void ChangeTimer(TimeSpan delay)
         private void CheckExpectedView(byte view_number)
         {
             if (context.ViewNumber == view_number) return;
+            if (context.State.HasFlag(ConsensusState.CommitSent))
+            {
+                if (context.State.HasFlag(ConsensusState.SignatureSent))
+                {
+                    // If signature was sent, we send again
+
+                    SignAndRelay(context.MakePrepareResponse(context.Signatures[context.MyIndex]));
```

Comment:
@shargon ，if a node (in commit phase) receives change view request, and re-send prepareResponse. From current code, the node (who have sent the changeView request ) will not accept prepareResponse because the viewNumber does not match. We may need to find a way to accept the re-send prepareResponse.

Reply:
Is just for dropped packets (network errors), maybe there are a disconnection, and this packet never arrive to your CN.